app/src/main/java/org/simple/clinic/summary/prescribeddrugs/DrugSummaryView.kt

```diff
@@ -13,6 +14,10 @@ import org.threeten.bp.format.DateTimeFormatter
 
 class DrugSummaryView(context: Context, attributeSet: AttributeSet) : CardView(context, attributeSet) {
 
+  init {
+    LayoutInflater.from(context).inflate(R.layout.patientsummary_drugview_content, this, true)
```

Comment:
Do we need to append `patientsummary` to this layout?
Since this is a separate view now, we can just called it `prescribed_drugs_summary_view`.

Reply:
Yeah, I did this so I could easily find it for autosuggesting when reading the layout. Will rename.

Reply:
Done.